console/src/helpers/Html.js

```diff
@@ -25,6 +25,10 @@ export default class Html extends Component {
       <html lang="en-us">
         <head>
           <link rel="icon" type="image/png" href="/rstatic/favicon.png" />
+          <link
```

Comment:
@karthikvt26 - this would require change in server and cli assets (index.html)

Reply:
Do you suggest any other way to solve this? bundling it in webpack and passing that too to the GraphiQL?

Reply:
Not used in the current release.